File: Examples/UI/WinForm/CSharp/TreeView/NTreeViewUC.cs
using System;
using System.Collections;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Windows.Forms;

using Nevron.UI.WinForm.Controls;
using Nevron.Examples.Framework.WinForm;

namespace Nevron.Examples.UI.WinForm
{
	/// <summary>
	/// Summary description for NTreeViewUC.
	/// </summary>
	public class NTreeViewUC : NExampleUserControl
	{
		#region Constructor

		public NTreeViewUC(MainForm f) : base(f)
		{
			InitializeComponent();

			Dock = DockStyle.Fill;
			nTreeView1.LabelEdit = true;
		}


		#endregion

		#region Implementation

		protected override void Dispose( bool disposing )
		{
			if( disposing )
			{
				if(components != null)
				{
					components.Dispose();
				}
			}
			base.Dispose(disposing);
		}

		public override void Initialize()
		{
			base.Initialize ();

			nTreeView1.ImageList = MainForm.TestImages;
			TreeNode node1, node2;

			for(int i = 0; i < 20; i++)
			{
				node1 = new TreeNode("NTreeNode "+i.ToString());
				node1.ImageIndex = i;
				node1.SelectedImageIndex = i;
				for(int j = 0; j < 20; j++)
				{
					node2 = new TreeNode("Sub-NTreeNode "+j.ToString());
					node2.ImageIndex = j;
					node2.SelectedImageIndex = j;
					node1.Nodes.Add(node2);
				}
				nTreeView1.Nodes.Add(node1);
			}

			++m_iSuspendUpdate;

			m_LineColorButton.Color = nTreeView1.LineColor;

			--m_iSuspendUpdate;
		}
		

		#endregion

		#region Event Handlers

		private void m_CheckBoxesCheck_CheckedChanged(object sender, System.EventArgs e)
		{
			nTreeView1.CheckBoxes = m_CheckBoxesCheck.Checked;
		}

		private void m_CustomDrawCheck_CheckedChanged(object sender, System.EventArgs e)
		{
			nTreeView1.CustomDraw = m_CustomDrawCheck.Checked;
		}

		private void m_BorderButton_Click(object sender, System.EventArgs e)
		{
			nTreeView1.Border.ShowEditor();
		}

		private void m_LineColorButton_ColorChanged(object sender, System.EventArgs e)
		{
			if(m_iSuspendUpdate != 0)
				return;

			nTreeView1.LineColor = m_LineColorButton.Color;
		}

		private void m_HideSelectionCheck_CheckedChanged(object sender, System.EventArgs e)
		{
			nTreeView1.HideSelection = m_HideSelectionCheck.Checked;
		}

		private void hotTrackCheck_CheckedChanged(object sender, System.EventArgs e)
		{
			nTreeView1.HotTracking = hotTrackCheck.Checked;
		}

		private void boldTextCheck_CheckedChanged(object sender, System.EventArgs e)
		{
			if(boldTextCheck.Checked)
			{
				nTreeView1.Font = new Font(nTreeView1.Font, FontStyle.Bold);
			}
			else
			{
				nTreeView1.Font = new Font(nTreeView1.Font, FontStyle.Regular);
			}
		}


		#endregion

		#region Component Designer generated code
		/// <summary> 
		/// Required method for Designer support - do not modify 
		/// the contents of this method with the code editor.
		/// </summary>
		private void InitializeComponent()
		{
			this.nTreeView1 = new Nevron.UI.WinForm.Controls.NTreeView();
			this.m_CheckBoxesCheck = new Nevron.UI.WinForm.Controls.NCheckBox();
			this.m_CustomDrawCheck = new Nevron.UI.WinForm.Controls.NCheckBox();
			this.m_BorderButton = new Nevron.UI.WinForm.Controls.NButton();
			this.m_LineColorButton = new Nevron.UI.WinForm.Controls.NColorButton();
			this.label1 = new System.Windows.Forms.Label();
			this.m_HideSelectionCheck = new Nevron.UI.WinForm.Controls.NCheckBox();
			this.hotTrackCheck = new Nevron.UI.WinForm.Controls.NCheckBox();
			this.boldTextCheck = new Nevron.UI.WinForm.Controls.NCheckBox();
			this.SuspendLayout();
			// 
			// nTreeView1
			// 
			this.nTreeView1.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom) 
				| System.Windows.Forms.AnchorStyles.Left) 
				| System.Windows.Forms.AnchorStyles.Right)));
			this.nTreeView1.BorderStyle = System.Windows.Forms.BorderStyle.None;
			this.nTreeView1.ImageIndex = -1;
			this.nTreeView1.Location = new System.Drawing.Point(8, 8);
			this.nTreeView1.Name = "nTreeView1";
			this.nTreeView1.SelectedImageIndex = -1;
			this.nTreeView1.Size = new System.Drawing.Size(424, 216);
			this.nTreeView1.TabIndex = 0;
			// 
			// m_CheckBoxesCheck
			// 
			this.m_CheckBoxesCheck.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
			this.m_CheckBoxesCheck.ButtonProperties.BorderOffset = 2;
			this.m_CheckBoxesCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft;
			this.m_CheckBoxesCheck.Location = new System.Drawing.Point(8, 232);
			this.m_CheckBoxesCheck.Name = "m_CheckBoxesCheck";
			this.m_CheckBoxesCheck.Size = new System.Drawing.Size(103, 24);
			this.m_CheckBoxesCheck.TabIndex = 5;
			this.m_CheckBoxesCheck.Text = "CheckBoxes";
			this.m_CheckBoxesCheck.CheckedChanged += new System.EventHandler(this.m_CheckBoxesCheck_CheckedChanged);
			// 
			// m_CustomDrawCheck
			// 
			this.m_CustomDrawCheck.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
			this.m_CustomDrawCheck.ButtonProperties.BorderOffset = 2;
			this.m_CustomDrawCheck.Checked = true;
			this.m_CustomDrawCheck.CheckState = System.Windows.Forms.CheckState.Checked;
			this.m_CustomDrawCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft;
			this.m_CustomDrawCheck.Location = new System.Drawing.Point(112, 232);
			this.m_CustomDrawCheck.Name = "m_CustomDrawCheck";
			this.m_CustomDrawCheck.Size = new System.Drawing.Size(103, 24);
			this.m_CustomDrawCheck.TabIndex = 6;
			this.m_CustomDrawCheck.Text = "Custom Draw";
			this.m_CustomDrawCheck.CheckedChanged += new System.EventHandler(this.m_CustomDrawCheck_CheckedChanged);
			// 
			// m_BorderButton
			// 
			this.m_BorderButton.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
			this.m_BorderButton.ImageAlign = System.Drawing.ContentAlignment.BottomCenter;
			this.m_BorderButton.Location = new System.Drawing.Point(120, 264);
			this.m_BorderButton.Name = "m_BorderButton";
			this.m_BorderButton.Size = new System.Drawing.Size(88, 24);
			this.m_BorderButton.TabIndex = 24;
			this.m_BorderButton.Text = "Border...";
			this.m_BorderButton.Click += new System.EventHandler(this.m_BorderButton_Click);
			// 
			// m_LineColorButton
			// 
			this.m_LineColorButton.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
			this.m_LineColorButton.ArrowClickOptions = false;
			this.m_LineColorButton.ArrowWidth = 14;
			this.m_LineColorButton.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft;
			this.m_LineColorButton.Location = new System.Drawing.Point(120, 296);
			this.m_LineColorButton.Name = "m_LineColorButton";
			this.m_LineColorButton.Size = new System.Drawing.Size(88, 24);
			this.m_LineColorButton.TabIndex = 25;
			this.m_LineColorButton.ColorChanged += new System.EventHandler(this.m_LineColorButton_ColorChanged);
			// 
			// label1
			// 
			this.label1.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
			this.label1.Location = new System.Drawing.Point(48, 296);
			this.label1.Name = "label1";
			this.label1.Size = new System.Drawing.Size(64, 23);
			this.label1.TabIndex = 26;
			this.label1.Text = "Line Color:";
			this.label1.TextAlign = System.Drawing.ContentAlignment.MiddleRight;
			// 
			// m_HideSelectionCheck
			// 
			this.m_HideSelectionCheck.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
			this.m_HideSelectionCheck.ButtonProperties.BorderOffset = 2;
			this.m_HideSelectionCheck.Checked = true;
			this.m_HideSelectionCheck.CheckState = System.Windows.Forms.CheckState.Checked;
			this.m_HideSelectionCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft;
			this.m_HideSelectionCheck.Location = new System.Drawing.Point(216, 232);
			this.m_HideSelectionCheck.Name = "m_HideSelectionCheck";
			this.m_HideSelectionCheck.Size = new System.Drawing.Size(103, 24);
			this.m_HideSelectionCheck.TabIndex = 27;
			this.m_HideSelectionCheck.Text = "Hide Selection";
			this.m_HideSelectionCheck.CheckedChanged += new System.EventHandler(this.m_HideSelectionCheck_CheckedChanged);
			// 
			// hotTrackCheck
			// 
			this.hotTrackCheck.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
			this.hotTrackCheck.ButtonProperties.BorderOffset = 2;
			this.hotTrackCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft;
			this.hotTrackCheck.Location = new System.Drawing.Point(320, 232);
			this.hotTrackCheck.Name = "hotTrackCheck";
			this.hotTrackCheck.Size = new System.Drawing.Size(103, 24);
			this.hotTrackCheck.TabIndex = 28;
			this.hotTrackCheck.Text = "Hot Track";
			this.hotTrackCheck.CheckedChanged += new System.EventHandler(this.hotTrackCheck_CheckedChanged);
			// 
			// boldTextCheck
			// 
			this.boldTextCheck.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
			this.boldTextCheck.ButtonProperties.BorderOffset = 2;
			this.boldTextCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft;
			this.boldTextCheck.Location = new System.Drawing.Point(320, 264);
			this.boldTextCheck.Name = "boldTextCheck";
			this.boldTextCheck.Size = new System.Drawing.Size(103, 24);
			this.boldTextCheck.TabIndex = 29;
			this.boldTextCheck.Text = "Bold Text";
			this.boldTextCheck.CheckedChanged += new System.EventHandler(this.boldTextCheck_CheckedChanged);
			// 
			// NTreeViewUC
			// 
			this.Controls.Add(this.boldTextCheck);
			this.Controls.Add(this.hotTrackCheck);
			this.Controls.Add(this.m_HideSelectionCheck);
			this.Controls.Add(this.label1);
			this.Controls.Add(this.m_LineColorButton);
			this.Controls.Add(this.m_BorderButton);
			this.Controls.Add(this.m_CustomDrawCheck);
			this.Controls.Add(this.m_CheckBoxesCheck);
			this.Controls.Add(this.nTreeView1);
			this.Name = "NTreeViewUC";
			this.Size = new System.Drawing.Size(440, 328);
			this.ResumeLayout(false);

		}
		#endregion

		#region Fields


		private System.ComponentModel.Container components = null;
		private Nevron.UI.WinForm.Controls.NButton m_BorderButton;
		private Nevron.UI.WinForm.Controls.NCheckBox m_CheckBoxesCheck;
		private Nevron.UI.WinForm.Controls.NCheckBox m_CustomDrawCheck;
		private Nevron.UI.WinForm.Controls.NColorButton m_LineColorButton;
		private System.Windows.Forms.Label label1;
		private Nevron.UI.WinForm.Controls.NCheckBox m_HideSelectionCheck;
		private Nevron.UI.WinForm.Controls.NCheckBox hotTrackCheck;
		private Nevron.UI.WinForm.Controls.NCheckBox boldTextCheck;
		private Nevron.UI.WinForm.Controls.NTreeView nTreeView1;

		#endregion
	}
}

File: Examples/UI/WinForm/VB.NET/TreeView/NTreeViewUC.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Windows.Forms

Imports Nevron.UI.WinForm.Controls
Imports Nevron.Examples.Framework.WinForm

Namespace Nevron.Examples.UI.WinForm
	''' <summary>
	''' Summary description for NTreeViewUC.
	''' </summary>
	Public Class NTreeViewUC
		Inherits NExampleUserControl
		#Region "Constructor"

		Public Sub New(ByVal f As MainForm)
			MyBase.New(f)
			InitializeComponent()

			Dock = DockStyle.Fill
			nTreeView1.LabelEdit = True
		End Sub


		#End Region

		#Region "Implementation"

		Protected Overrides Overloads Sub Dispose(ByVal disposing As Boolean)
			If disposing Then
				If Not components Is Nothing Then
					components.Dispose()
				End If
			End If
			MyBase.Dispose(disposing)
		End Sub

		Public Overrides Sub Initialize()
			MyBase.Initialize ()

			nTreeView1.ImageList = MainForm.TestImages
			Dim node1, node2 As TreeNode

			For i As Integer = 0 To 19
				node1 = New TreeNode("NTreeNode " & i.ToString())
				node1.ImageIndex = i
				node1.SelectedImageIndex = i
				For j As Integer = 0 To 19
					node2 = New TreeNode("Sub-NTreeNode " & j.ToString())
					node2.ImageIndex = j
					node2.SelectedImageIndex = j
					node1.Nodes.Add(node2)
				Next j
				nTreeView1.Nodes.Add(node1)
			Next i

			m_iSuspendUpdate += 1

			m_LineColorButton.Color = nTreeView1.LineColor

			m_iSuspendUpdate -= 1
		End Sub


		#End Region

		#Region "Event Handlers"

		Private Sub m_CheckBoxesCheck_CheckedChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles m_CheckBoxesCheck.CheckedChanged
			nTreeView1.CheckBoxes = m_CheckBoxesCheck.Checked
		End Sub

		Private Sub m_CustomDrawCheck_CheckedChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles m_CustomDrawCheck.CheckedChanged
			nTreeView1.CustomDraw = m_CustomDrawCheck.Checked
		End Sub

		Private Sub m_BorderButton_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles m_BorderButton.Click
			nTreeView1.Border.ShowEditor()
		End Sub

		Private Sub m_LineColorButton_ColorChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles m_LineColorButton.ColorChanged
			If m_iSuspendUpdate <> 0 Then
				Return
			End If

			nTreeView1.LineColor = m_LineColorButton.Color
		End Sub

		Private Sub m_HideSelectionCheck_CheckedChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles m_HideSelectionCheck.CheckedChanged
			nTreeView1.HideSelection = m_HideSelectionCheck.Checked
		End Sub

		Private Sub hotTrackCheck_CheckedChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles hotTrackCheck.CheckedChanged
			nTreeView1.HotTracking = hotTrackCheck.Checked
		End Sub

		Private Sub boldTextCheck_CheckedChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles boldTextCheck.CheckedChanged
			If boldTextCheck.Checked Then
				nTreeView1.Font = New Font(nTreeView1.Font, FontStyle.Bold)
			Else
				nTreeView1.Font = New Font(nTreeView1.Font, FontStyle.Regular)
			End If
		End Sub


		#End Region

		#Region "Component Designer generated code"
		''' <summary> 
		''' Required method for Designer support - do not modify 
		''' the contents of this method with the code editor.
		''' </summary>
		Private Sub InitializeComponent()
			Me.nTreeView1 = New Nevron.UI.WinForm.Controls.NTreeView()
			Me.m_CheckBoxesCheck = New Nevron.UI.WinForm.Controls.NCheckBox()
			Me.m_CustomDrawCheck = New Nevron.UI.WinForm.Controls.NCheckBox()
			Me.m_BorderButton = New Nevron.UI.WinForm.Controls.NButton()
			Me.m_LineColorButton = New Nevron.UI.WinForm.Controls.NColorButton()
			Me.label1 = New System.Windows.Forms.Label()
			Me.m_HideSelectionCheck = New Nevron.UI.WinForm.Controls.NCheckBox()
			Me.hotTrackCheck = New Nevron.UI.WinForm.Controls.NCheckBox()
			Me.boldTextCheck = New Nevron.UI.WinForm.Controls.NCheckBox()
			Me.SuspendLayout()
			' 
			' nTreeView1
			' 
			Me.nTreeView1.Anchor = (CType((((System.Windows.Forms.AnchorStyles.Top Or System.Windows.Forms.AnchorStyles.Bottom) Or System.Windows.Forms.AnchorStyles.Left) Or System.Windows.Forms.AnchorStyles.Right), System.Windows.Forms.AnchorStyles))
			Me.nTreeView1.BorderStyle = System.Windows.Forms.BorderStyle.None
			Me.nTreeView1.ImageIndex = -1
			Me.nTreeView1.Location = New System.Drawing.Point(8, 8)
			Me.nTreeView1.Name = "nTreeView1"
			Me.nTreeView1.SelectedImageIndex = -1
			Me.nTreeView1.Size = New System.Drawing.Size(424, 216)
			Me.nTreeView1.TabIndex = 0
			' 
			' m_CheckBoxesCheck
			' 
			Me.m_CheckBoxesCheck.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Left), System.Windows.Forms.AnchorStyles))
			Me.m_CheckBoxesCheck.ButtonProperties.BorderOffset = 2
			Me.m_CheckBoxesCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft
			Me.m_CheckBoxesCheck.Location = New System.Drawing.Point(8, 232)
			Me.m_CheckBoxesCheck.Name = "m_CheckBoxesCheck"
			Me.m_CheckBoxesCheck.Size = New System.Drawing.Size(103, 24)
			Me.m_CheckBoxesCheck.TabIndex = 5
			Me.m_CheckBoxesCheck.Text = "CheckBoxes"
'			Me.m_CheckBoxesCheck.CheckedChanged += New System.EventHandler(Me.m_CheckBoxesCheck_CheckedChanged);
			' 
			' m_CustomDrawCheck
			' 
			Me.m_CustomDrawCheck.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Left), System.Windows.Forms.AnchorStyles))
			Me.m_CustomDrawCheck.ButtonProperties.BorderOffset = 2
			Me.m_CustomDrawCheck.Checked = True
			Me.m_CustomDrawCheck.CheckState = System.Windows.Forms.CheckState.Checked
			Me.m_CustomDrawCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft
			Me.m_CustomDrawCheck.Location = New System.Drawing.Point(112, 232)
			Me.m_CustomDrawCheck.Name = "m_CustomDrawCheck"
			Me.m_CustomDrawCheck.Size = New System.Drawing.Size(103, 24)
			Me.m_CustomDrawCheck.TabIndex = 6
			Me.m_CustomDrawCheck.Text = "Custom Draw"
'			Me.m_CustomDrawCheck.CheckedChanged += New System.EventHandler(Me.m_CustomDrawCheck_CheckedChanged);
			' 
			' m_BorderButton
			' 
			Me.m_BorderButton.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Left), System.Windows.Forms.AnchorStyles))
			Me.m_BorderButton.ImageAlign = System.Drawing.ContentAlignment.BottomCenter
			Me.m_BorderButton.Location = New System.Drawing.Point(120, 264)
			Me.m_BorderButton.Name = "m_BorderButton"
			Me.m_BorderButton.Size = New System.Drawing.Size(88, 24)
			Me.m_BorderButton.TabIndex = 24
			Me.m_BorderButton.Text = "Border..."
'			Me.m_BorderButton.Click += New System.EventHandler(Me.m_BorderButton_Click);
			' 
			' m_LineColorButton
			' 
			Me.m_LineColorButton.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Left), System.Windows.Forms.AnchorStyles))
			Me.m_LineColorButton.ArrowClickOptions = False
			Me.m_LineColorButton.ArrowWidth = 14
			Me.m_LineColorButton.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft
			Me.m_LineColorButton.Location = New System.Drawing.Point(120, 296)
			Me.m_LineColorButton.Name = "m_LineColorButton"
			Me.m_LineColorButton.Size = New System.Drawing.Size(88, 24)
			Me.m_LineColorButton.TabIndex = 25
'			Me.m_LineColorButton.ColorChanged += New System.EventHandler(Me.m_LineColorButton_ColorChanged);
			' 
			' label1
			' 
			Me.label1.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Left), System.Windows.Forms.AnchorStyles))
			Me.label1.Location = New System.Drawing.Point(48, 296)
			Me.label1.Name = "label1"
			Me.label1.Size = New System.Drawing.Size(64, 23)
			Me.label1.TabIndex = 26
			Me.label1.Text = "Line Color:"
			Me.label1.TextAlign = System.Drawing.ContentAlignment.MiddleRight
			' 
			' m_HideSelectionCheck
			' 
			Me.m_HideSelectionCheck.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Left), System.Windows.Forms.AnchorStyles))
			Me.m_HideSelectionCheck.ButtonProperties.BorderOffset = 2
			Me.m_HideSelectionCheck.Checked = True
			Me.m_HideSelectionCheck.CheckState = System.Windows.Forms.CheckState.Checked
			Me.m_HideSelectionCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft
			Me.m_HideSelectionCheck.Location = New System.Drawing.Point(216, 232)
			Me.m_HideSelectionCheck.Name = "m_HideSelectionCheck"
			Me.m_HideSelectionCheck.Size = New System.Drawing.Size(103, 24)
			Me.m_HideSelectionCheck.TabIndex = 27
			Me.m_HideSelectionCheck.Text = "Hide Selection"
'			Me.m_HideSelectionCheck.CheckedChanged += New System.EventHandler(Me.m_HideSelectionCheck_CheckedChanged);
			' 
			' hotTrackCheck
			' 
			Me.hotTrackCheck.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Left), System.Windows.Forms.AnchorStyles))
			Me.hotTrackCheck.ButtonProperties.BorderOffset = 2
			Me.hotTrackCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft
			Me.hotTrackCheck.Location = New System.Drawing.Point(320, 232)
			Me.hotTrackCheck.Name = "hotTrackCheck"
			Me.hotTrackCheck.Size = New System.Drawing.Size(103, 24)
			Me.hotTrackCheck.TabIndex = 28
			Me.hotTrackCheck.Text = "Hot Track"
'			Me.hotTrackCheck.CheckedChanged += New System.EventHandler(Me.hotTrackCheck_CheckedChanged);
			' 
			' boldTextCheck
			' 
			Me.boldTextCheck.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Left), System.Windows.Forms.AnchorStyles))
			Me.boldTextCheck.ButtonProperties.BorderOffset = 2
			Me.boldTextCheck.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft
			Me.boldTextCheck.Location = New System.Drawing.Point(320, 264)
			Me.boldTextCheck.Name = "boldTextCheck"
			Me.boldTextCheck.Size = New System.Drawing.Size(103, 24)
			Me.boldTextCheck.TabIndex = 29
			Me.boldTextCheck.Text = "Bold Text"
'			Me.boldTextCheck.CheckedChanged += New System.EventHandler(Me.boldTextCheck_CheckedChanged);
			' 
			' NTreeViewUC
			' 
			Me.Controls.Add(Me.boldTextCheck)
			Me.Controls.Add(Me.hotTrackCheck)
			Me.Controls.Add(Me.m_HideSelectionCheck)
			Me.Controls.Add(Me.label1)
			Me.Controls.Add(Me.m_LineColorButton)
			Me.Controls.Add(Me.m_BorderButton)
			Me.Controls.Add(Me.m_CustomDrawCheck)
			Me.Controls.Add(Me.m_CheckBoxesCheck)
			Me.Controls.Add(Me.nTreeView1)
			Me.Name = "NTreeViewUC"
			Me.Size = New System.Drawing.Size(440, 328)
			Me.ResumeLayout(False)

		End Sub
		#End Region

		#Region "Fields"


		Private components As System.ComponentModel.Container = Nothing
		Private WithEvents m_BorderButton As Nevron.UI.WinForm.Controls.NButton
		Private WithEvents m_CheckBoxesCheck As Nevron.UI.WinForm.Controls.NCheckBox
		Private WithEvents m_CustomDrawCheck As Nevron.UI.WinForm.Controls.NCheckBox
		Private WithEvents m_LineColorButton As Nevron.UI.WinForm.Controls.NColorButton
		Private label1 As System.Windows.Forms.Label
		Private WithEvents m_HideSelectionCheck As Nevron.UI.WinForm.Controls.NCheckBox
		Private WithEvents hotTrackCheck As Nevron.UI.WinForm.Controls.NCheckBox
		Private WithEvents boldTextCheck As Nevron.UI.WinForm.Controls.NCheckBox
		Private nTreeView1 As Nevron.UI.WinForm.Controls.NTreeView

		#End Region
	End Class
End Namespace
